File: ruby/iniciante/1010.rb
num1 = gets.chomp.split(' ')
p1 = num1[1].to_i
v1 = num1[2].to_f
num2 = gets.chomp.split(' ')
p2 = num2[1].to_i
v2 = num2[2].to_f
puts "VALOR A PAGAR: R$ "'%.2f' % "#{(p1 * v1) + (p2 * v2)}"

File: python3/iniciante/1010.py
p1 = input().split()
p2 = input().split()

total = float(p1[1]) * float(p1[2]) + float(p2[1]) * float(p2[2])

total = "{:.2f}".format(total)

print("VALOR A PAGAR: R$ " + total)
